ospfd/ospf_vty.c

```diff
@@ -12952,6 +12952,53 @@ DEFUN (clear_ip_ospf_interface,
 	return CMD_SUCCESS;
 }
 
+DEFPY_HIDDEN(ospf_lsa_refresh_timer, ospf_lsa_refresh_timer_cmd,
+	     "[no$no] ospf lsa-refresh [(120-1800)]$value",
+	     NO_STR
+	     "OSPF protocol\n"
+	     "OSPF lsa refresh timer\n"
+	     "timer value in seconds\n")
+{
+	VTY_DECLVAR_INSTANCE_CONTEXT(ospf, ospf)
+
+	if (no) {
+		ospf->lsa_refresh_timer = OSPF_LS_REFRESH_TIME;
+	} else {
+		if ((value < 120) || (value > 1800)) {
+			vty_out(vty, "Invalid timer value.\r\n ");
```

Comment:
I don't think control reaches  here , DELPY will implicitly take care of this boundary validation, can you please check ?

Reply:
checked this, boundary validations are not needed, removed these checks.